### ISSUE TITLE
pandoc-citerproc.exe: could not find locale data for DE-

### ISSUE THREAD

**Issue by PhilippSchweizer** [jgm/pandoc-citeproc#222]
Hi,
I am trying to convert from .md to .pdf. I use a .bib file and this is why I type the following in to the console:
`pandoc --filter pandoc-citeproc input.md -s -o output.pdf`

Then it throws me an error:
`pandoc-citeproc.exe: could not find locale data for DE-`
`pandoc.exe: Error running filter pandoc-citeproc`

Windows 7
pandoc version 1.17.0.2
citeproc version 0.9.1.1

Any help is appreciated. Thanks.

**Comment by ousia** [comment]
@PhilippSchweizer, I guess this is a duplicate from jgm/pandoc#2838.

**Comment by ousia** [comment]
Sorry, on a closer inspection, which one is the language variable in your document: `de` or `de-CH`?

**Comment by PhilippSchweizer** [comment]
Looked at the issue you referenced and i am confident that the information there will help me fix my problem (can't try now though).
To your question: I encountered the problem even when i used no language variable at all in my document. I then used de-DE and the problem persisted. But I didn't try `--metadata lang=de-DE` which is the first I will do bevor setting or deleting any environment variables.
